roles/ovirt-engine-setup/tasks/main.yml

```diff
@@ -0,0 +1,69 @@
+---
+# health page
+- name: check if ovirt-engine running (health page)
+  uri:
+    url: "http://{{ ansible_fqdn }}/ovirt-engine/services/health"
+    status_code: 200
+  register: ovirt_engine_status
+  retries: 2
+  delay: 5
+  until: ovirt_engine_status|success
+  ignore_errors: True
+
+# copy default answer file
+- name: copy default answerfile
+  template:
+    src: answerfile_{{ ovirt_engine_version }}_basic.txt.j2
+    dest: /tmp/answerfile.txt
+    mode: 0644
```

Comment:
Any reason this is world readable if there are stored passwords in plain text?

Reply:
no idea. 
but the engine setup should read this file. so it should be plain text and readable. ... 
do u suggest to make it 0600 ?

Reply:
yes, make it 0600 and set no_log to true, as I think it's printed to stdout as well

Reply:
ok

Reply:
What is the credentials of the user running this Ansible script? You do NOT put files with predicted name under /tmp (especially if you are root!).